plugins/modules/junos_static_route.py

```diff
@@ -168,8 +213,8 @@ def main():
     """ main entry point for module execution
     """
     element_spec = dict(
-        address=dict(aliases=["prefix"]),
-        next_hop=dict(),
+        address=dict(aliases=["prefix"], required=True),
```

Comment:
Top level keys cannot be `required` for modules having `aggregate` option

Reply:
Done